src/core/crypto_tls.c

```diff
@@ -608,7 +617,11 @@ QuicCryptoTlsReadInitial(
                 Connection,
                 Buffer + TLS_MESSAGE_HEADER_LENGTH,
                 MessageLength,
-                Info);
+                Info
+#ifdef QUIC_TLS_SECRETS_SUPPORT
+                , Connection->TlsSecrets
+#endif
```

Comment:
Nothing in these functions ever reads from `Connection` (I'm pretty sure). It's only passed in for logging reasons. I'd prefer to keep it that way, so we could more easily unit test these functions in the future. So can we just pass in this (optional) pointer please to `QuicCryptoTlsReadInitial`?